setup.py

```diff
@@ -36,7 +37,7 @@
         'orderedset>=2.0.3',
         'phonenumbers~=8.12.12',
         'pypdf2 ~= 3.0.1',
-        'python-json-logger>=0.1.11',
+        'python-json-logger~=2.0.7',
```

Comment:
WOW, what an update.

Reply:
Note the "greater than."  I just made it match what is already in use.

Reply:
It was just shocking that it hadn't been coded to anything newer.